gcp_variant_transforms/testing/integration/small_tests/valid_4_2_VEP.json

```diff
@@ -4,12 +4,16 @@
   "input_pattern": "gs://gcp-variant-transforms-testfiles/small_tests/valid-4.2_VEP.vcf",
   "annotation_field": "CSQ",
   "runner": "DataflowRunner",
-  "validation_query": [
-    "SELECT COUNT(DISTINCT CSQ.Feature) AS num_features ",
-    "FROM {TABLE_NAME} AS t, t.alternate_bases as alts, alts.CSQ as CSQ ",
-    "WHERE start_position = 1110695 AND alts.alt = 'G'"
-  ],
-  "expected_query_result": {
-    "num_features": 3
-  }
+  "assertion_configs": [
+    {
```

Comment:
Now that we have the support, please add a second query to this config, just checking the aggregates like all other tests.

Reply:
Correct. One of the next tasks is to design more test cases.

Reply:
Agreed, but still it would have been nice to have at least one case that it does multiple queries in this very PR (specially because this one lacked the simple query in every other test). Anyways, up to you.

Reply:
Done.